packages/core/src/session.ts

```diff
@@ -89,8 +91,11 @@ class Session {
    * @param {boolean} args.reactive - Whether this session should create reactive streams
    * @param {number} args.fetchSize - Defines how many records is pulled in each pulling batch
    * @param {string} args.impersonatedUser - The username which the user wants to impersonate for the duration of the session.
-   * @param {AuthToken} args.auth - the target auth for the to-be-acquired connection
+   * @param {BookmarkManager} args.bookmarkManager - The bookmark manager used for this session.
    * @param {NotificationFilter} args.notificationFilter - The notification filter used for this session.
+   * @param {AuthToken} args.auth - the target auth for the to-be-acquired connection
+   * @param {Logger} args.log - the logger used for logs in this session.
+   * @param {(impersonatedUser:string, user:string database:string) => void} args.homeDatabaseCallback - callback used to update the home database cache
```

Comment:
```suggestion
   * @param {(user: string, database: string) => void} args.homeDatabaseCallback - callback used to update the home database cache
```
Judging by the type below.